addOns/ascanrules/src/main/resources/org/zaproxy/zap/extension/ascanrules/resources/Messages.properties

```diff
@@ -129,25 +127,13 @@ ascanrules.sqlinjection.alert.timebased.extrainfo=The query time is controllable
 ascanrules.sqlinjection.authbypass.name=SQL Injection - Authentication Bypass
 ascanrules.sqlinjection.authbypass.desc=SQL injection may be possible on a login page, potentially allowing the application's authentication mechanism to be bypassed 
 
-# TODO some duplicates
```

Comment:
`ascanrules.sqlinjection.mssql.soln` and `ascanrules.sqlinjection.mssql.desc` are still duplicating the main ones.